src/global_helpers.R

```diff
@@ -749,3 +750,55 @@ update_product_statuses <- function(network, domain){
 
     return()
 }
+
+unit_converter <- function(val, input_unit, output_unit) {
```

Comment:
@spencerrhea next time we screenshare, let's see this puppy in action. @matthewross07, do you have a go-to unit conversion package?